CMakeLists.txt

```diff
@@ -104,12 +104,16 @@ endif ()
 
 if (ENABLE_LIBPNG)
     if (MSVC)
-        find_package(libpng CONFIG REQUIRED)
+        find_package(PNG CONFIG REQUIRED)
```

Comment:
Perhaps due to this change?

Reply:
Now it is, but it started before it was introduced. Just rebuilding the project would probably trigger it again

Reply:
I don't know then, I haven't maintained this for a long time. It is possible something has changed with how you include the package.